File: benchmark/fib.rb
#!/usr/bin/env ruby
#
def fib(n)
  if n == 0 || n == 1
    n
  else
    fib(n-1) + fib(n-2)
  end
end

36.times { |i| puts "n=#{i} => #{fib(i)}" }

File: benchmark/fib.py
#!/usr/bin/env python

def fib(n):
  if n == 0 or n == 1:
    return n
  else:
    return fib(n-1) + fib(n-2)

for i in range(36):
  print("n=%d => %d" % (i, fib(i)))

